wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-meta-info.php

```diff
@@ -0,0 +1,74 @@
+		<a href="<?php echo esc_url( get_my_courses_page_url() ); ?>">
+			<?php esc_html_e( 'All My Courses', 'wporg-learn' ); ?>
+		</a>
+	</p>
+	<!-- /wp:paragraph -->
+	<?php endif; ?>
+
+	<!-- wp:sensei-lms/course-progress {"defaultBarColor":"blueberry-1"} /-->
+
+	<!-- wp:sensei-lms/button-take-course {"align":"full","borderRadius":2,"className":"is-style-default"} -->
+	<div class="wp-block-sensei-lms-button-take-course is-style-default wp-block-sensei-button wp-block-button has-text-align-full">
+		<button class="wp-block-button__link" style="border-radius:2px">
+			<?php esc_html_e( 'Take this Course', 'wporg-learn' ); ?>
+		</button>
+	</div>
+	<!-- /wp:sensei-lms/button-take-course -->
+
+	<!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center"},"className":""wporg-learn-sidebar-playground"} -->
+	<div class="wp-block-buttons">
+		<!-- wp:button {"textAlign":"center","width":100,"style":{"border":{"radius":"2px"},"spacing":{"padding":{"left":"13px","right":"13px","top":"16px","bottom":"16px"}},"typography":{"lineHeight":0,"fontStyle":"normal","fontWeight":"400"}},"className":"aligncenter is-style-outline","fontSize":"normal","fontFamily":"inter"} -->
+		<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size aligncenter is-style-outline has-inter-font-family has-normal-font-size wporg-learn-sidebar-playground" style="font-style:normal;font-weight:400;line-height:0">
+			<a class="wp-block-button__link has-text-align-center wp-element-button" href="https://wordpress.org/playground/demo/?step=playground&amp;theme=twentytwentythree" style="border-radius:2px;padding-top:16px;padding-right:13px;padding-bottom:16px;padding-left:13px" target="_blank" rel="noreferrer noopener">
+				<?php esc_html_e( 'Practice on a private demo site', 'wporg-learn' ); ?>
+			</a>
+		</div>
+		<!-- /wp:button -->
+	</div>
+	<!-- /wp:buttons -->
+
+	<!-- wp:wporg-learn/sensei-meta-list {"type":"course","fontSize":"normal"} /-->
+
+	<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"400","lineHeight":"34px"}},"fontSize":"large","fontFamily":"inter"} -->
+	<p class="has-inter-font-family has-large-font-size" id="suggestions" style="font-style:normal;font-weight:400;line-height:34px">
+		<?php esc_html_e( 'Suggestions', 'wporg-learn' ); ?>
+	</p>
+	<!-- /wp:paragraph -->
+
+	<!-- wp:paragraph {"style":{"typography":{"fontStyle":"normal","fontWeight":"400,"lineHeight":"26px"},"spacing":{"margin":{"bottom":"40px"}}},"fontSize":"normal","fontFamily":"inter"} -->
+	<p class="has-inter-font-family has-normal-font-size" style="margin-bottom:40px;font-style:normal;font-weight:400;line-height:26px">
+		<?php esc_html_e( 'Found a typo, grammar error or outdated screenshot? ', 'wporg-learn' ); ?><a href="https://learn.wordpress.org/report-content-errors/"><?php esc_html_e( 'Contact us', 'wporg-learn' ); ?></a>
```

Comment:
This link is 404 for me, maybe it should be https://learn.wordpress.org/report-content-feedback/ ?

Reply:
I see that's the current link on prod, seems to be a redirect or something

Reply:
weird, it works on my end, just it gets redirected to the link you showed.
https://github.com/WordPress/Learn/assets/18050944/84f30d65-383f-4728-91bd-82d87fcfaf81

c9a2064